utils/fetchIssuedAssets.ts

```diff
@@ -0,0 +1,60 @@
+import ERC725js, { ERC725JSONSchema } from '@erc725/erc725.js';
+import LSP12Schemas from '@erc725/erc725.js/schemas/LSP12IssuedAssets.json';
+import LSP4Schemas from '@erc725/erc725.js/schemas/LSP4DigitalAsset.json';
+
+const verifyCreatorShip = async (
+  ownerAddress: string,
+  provider: any,
+  issuedAssetAddress: string,
+): Promise<string> => {
+  const erc725Asset = new ERC725js(
+    LSP4Schemas as ERC725JSONSchema[],
+    issuedAssetAddress,
+    provider,
+  );
+  try {
+    const fetchCreators = await erc725Asset.fetchData('LSP4Creators[]');
+    const creators = fetchCreators.value as string[];
+    creators.filter((creator) => {
+      if (creator === ownerAddress) {
+        return creator;
+      }
+    });
+    return '';
```

Comment:
Might be easier to deal with either a string or `null`. But for that function I think you just want to return true if the owner is also a creator?